_components/search-bar.md

```diff
@@ -8,55 +8,40 @@ title: Search Bar
 
   <div class="usa-grid">
     <div class="usa-width-one-half">
-      <form class="usa-search usa-search-big">           
-        <fieldset>
```

Comment:
@msecret I think it's more than just a single field bc we have a button in the group as well. 

These were previous approved by @nickbristow with fieldsets so if we're removing it, we need to run this by him and make sure it's ok.